src/object.js

```diff
@@ -127,6 +127,15 @@ define([], function() {
 		return allSubclasses;
 	};
 
+	/**
+	 * Return all concrete subclasses.
+	 */
+	object.allConcreteSubclasses = function() {
+		return this.allSubclasses().filter(function(klass) {
+			return !klass.isAbstract;
```

Comment:
the iterating method can also be extracted and reused, so it encapsulates the behavior
and avoid iterating twice